java/gateway-core/src/main/java/com/avaulta/gateway/rules/transforms/Transform.java

```diff
@@ -217,6 +217,13 @@ public FilterTokenByRegex clone() {
     @Getter
     public static class PseudonymizeEmailHeader extends Transform {
 
+        /**
+         * how to encode to the resulting pseudonym
+         */
+        @JsonInclude(JsonInclude.Include.NON_DEFAULT) //doesn't work for enums ...
+        @Builder.Default
+        PseudonymEncoder.Implementations encoding = PseudonymEncoder.Implementations.JSON;
```

Comment:
I guess by default should be URL_SAFE_TOKEN

Reply:
yeah, technically breaking change to behavior - but might as well. no one but us should be dependent on the format.